src/GovUk.Education.ExploreEducationStatistics.Data.Model/Filter.cs

```diff
@@ -21,12 +22,17 @@ public Filter()
         {
         }
 
-        public Filter(string? hint, string label, string name, Guid subjectId)
+        public Filter(string? hint,
+            string label,
+            string name,
+            string groupCsvColumn,
```

Comment:
Just wondering if the type of `groupCsvColumn` should be a nullable string to match the model?

Reply:
Good point, I think I missed this because `MetaRow` didn't have the nullable annotation context enabled and doing so just now then highlighted this problem and some other related problems. Fixed them in dfe1afc.